contracts/StrategySushiswapPair.sol

```diff
@@ -131,15 +131,16 @@ contract StrategySushiswapPair is BaseStrategy {
      * strategy and reduce it's overall position if lower than expected returns
      * are sustained for long periods of time.
      */
-    function prepareReturn() internal override {
-        reserve = want.balanceOf(address(this)).sub(outstanding);
+    function prepareReturn(uint256 _debtOutstanding) internal override returns (uint256 _profit){
+        setReserve(want.balanceOf(address(this)).sub(_debtOutstanding));
         SushiChef(chef).deposit(pid, 0);
         uint _amount = IERC20(reward).balanceOf(address(this));
-        if (_amount < 1 gwei) return;
+        if (_amount < 1 gwei) return 0;
         swap(reward, token0, _amount / 2);
         _amount = IERC20(reward).balanceOf(address(this));
         swap(reward, token1, _amount);
         add_liquidity();
+        _profit = want.balanceOf(address(this)).sub(getReserve()).sub(_debtOutstanding);
```

Comment:
mmmmm interesting.
I am doing `_profit = balanceOfWant().sub(getReserve());` without considering `_debtOutstanding`.

Reply:
yep debt outstanding does not impact profit. just reserve

Reply:
Ah good to know. Thanks